tests/10-vcdm2.js

```diff
@@ -299,14 +299,10 @@ describe('Verifiable Credentials Data Model v2.0', function() {
         await assert.rejects(
           issue(require('./input/credential-issuer-object-no-id-fail.json')));
       });
-      it2('A credential MUST have an validFrom property.', async function() {
-        await assert.rejects(
-          issue(require('./input/credential-no-validfrom-fail.json')));
-      });
-      it2('The value of the validFrom property MUST be a string value of ' +
-        'an [XMLSCHEMA11-2] combined date-time string representing the date' +
-        ' and time the credential becomes valid, which could be a date and' +
-        ' time in the future.', async function() {
+      it2('If present, The value of the validFrom property MUST be a string' +
+        ' value of an [XMLSCHEMA11-2] combined date-time string representing' +
+        ' the date and time the credential becomes valid, which could be a' +
+        ' date and time in the future.', async function() {
```

Comment:
Note that this normative statement has changed:

https://w3c.github.io/vc-data-model/#validity-period

If you're updating it now, we might as well update it to the current text in the spec.

Reply:
Addressed here: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/19/commits/18c9c7db40591f794a017376875314955cf38e7b